feat: update readme (#1)
eve-autonomy/ad_sound.default: README.md

> # autonomous_driving_sound

Reviewer: ```suggestion
# Set of sound files for autonomous driving notification
```

## Overview

> This is a package for managing parameters to make [ad_sound_manager](https://github.com/eve-autonomy/ad_sound_manager) easier to use.

Reviewer: ```suggestion
This is a default package of sound files for [ad_sound_manager](https://github.com/eve-autonomy/ad_sound_manager/).
```

> Users can place and use user-defined directories in `wavs` directory.

Reviewer: ```suggestion
Users can place your own sound files in a user-defined directories under sub-directory `wavs`.
```

Since this repository is a template, it contains only the `default` directory.

> The list of names of sound files managed in the use-defined directory should be aligned with those in the `default` directory.

Reviewer: ```suggestion
Sound file names in config have to be aligned with those file names in the `default` directory.
```

## List of supported files
See the [parameter description for ad_sound_manager](https://github.com/eve-autonomy/ad_sound_manager#parameter-description).

## Extensibility of this package

> This package is a template as a parameter configuration.
> 
> If you want to change sound files in this package and use it, follow the steps below.

Reviewer: ```suggestion
This package is a set of template default sounds.
If you want to use your own sound files, please see the following steps;
```

> 1. Fork this repository to create a new one.

Reviewer: ```suggestion
1. Fork this repository.
```

1. Create a new directory in the [wavs](#/wavs) directory and insert the new audio file in it.

> 1. Change the value of `lang`, which is the [launch arguments of ad_sound_manager](https://github.com/eve-autonomy/ad_sound_manager#launch-arguments), to the new directory name.

Reviewer: ```suggestion
1. Change the value `lang`, which works as a new directory name in [launch arguments of ad_sound_manager](https://github.com/eve-autonomy/ad_sound_manager#launch-arguments).
```
